packages/eslint-config-react/index.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   extends: ['plugin:react/recommended', '@zonedigital/eslint-config-zone'],
   plugins: ['react-hooks'],
   rules: {
+    'react-hooks/exhaustive-deps': 'warn',
```

Comment:
I've added this as a warning for the moment as, although released and stable it appears the output is still under review. Kinda feel like we should have this as an error in the long term as not doing the hook dependencies fully will cause needless re-rending/processing.

Reply:
@sburrows-zone